src/test/java/com/exasol/adapter/dialects/exasol/ExasolSqlGenerationVisitorTest.java

```diff
@@ -44,6 +44,6 @@ void testVisitSqlLiteralTimestampUtc() {
     void testVisitSqlLiteralDouble() {
         final ExasolSqlGenerationVisitor exasolSqlGenerationVisitor = new ExasolSqlGenerationVisitor(
                 this.exasolSqlDialect, null);
-        assertThat(exasolSqlGenerationVisitor.visit(new SqlLiteralDouble(1.23)), equalTo("CAST(1.23 AS DOUBLE)"));
+        assertThat(exasolSqlGenerationVisitor.visit(new SqlLiteralDouble(1.23)), equalTo("CAST(1.23E0 AS DOUBLE)"));
```

Comment:
From reading #49, is it only requires exponent with zero `E0`? I think, the sql visitor is already going to generate correct exponentiation, right?

Reply:
yes